crime-applications-adaptor/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/testutils/MockWebServerStubs.java

```diff
@@ -11,19 +11,23 @@ public class MockWebServerStubs {
 
     private static final String REQUEST_PATH_EFORM_STAGING_WITH_NO_MAAT_REF = "/initialise/6000308";
     private static final String REQUEST_PATH_EFORM_STAGING_WITH_MAAT_REF = "/initialise/6000309";
-    private static final String REQUEST_PATH_CRIME_APPLY_DATASTORE = "/6000308";
-    private static final String REQUEST_PATH_CRIME_APPLY_4XX = "/403";
+    private static final String REQUEST_PATH_EFORM_STAGING_HUB_USER = "/initialise/6000310";
     private static final String REQUEST_PATH_EFORM_STAGING_4XX = "/initialise/403";
+    private static final String REQUEST_PATH_CRIME_APPLY_DATASTORE_1 = "/6000308";
```

Comment:
Can you make these variables `REQUEST_PATH_CRIME_APPLY_DATASTORE_1` and `REQUEST_PATH_CRIME_APPLY_DATASTORE_2` more descriptive? Maybe indicate that they are 200 OK responses and correspond to USN 6000308 for an example.

Reply:
New filenames look good